docs/deprecated.md

```diff
@@ -112,6 +113,22 @@ The table below provides an overview of the current status of deprecated feature
 | Removed    | [`--run` flag on `docker commit`](#--run-flag-on-docker-commit)                                                                    | v0.10      | v1.13  |
 | Removed    | [Three arguments form in `docker import`](#three-arguments-form-in-docker-import)                                                  | v0.6.7     | v1.12  |
 
+### API CORS headers
+
+**Deprecated in Release: v27.0**
+**Target For Removal In Release: v28.0**
+
+Daemon's `api-cors-header` flag is deemed insecure as it could be enabled
+without any Authz plugin enabled beforehand.
```

Comment:
Wondering if we can make this description more generic; Authz plugins are _one way_ to do this, but are also a bit of a corner-case use (there may be many other approaches). The `authz plugin` feature also isn't very actively maintained, so somewhat trying to avoid it coming across as a recommendation.

cc @dvdksn in case you have good suggestions.

Reply:
could we just pretend Authz was short for authorization?

```suggestion
Daemon's `api-cors-header` flag is deemed insecure as it could be enabled
without any authorization plugin enabled beforehand.
```

Reply:
Thanks @dvdksn!